core/src/main/java/com/arcrobotics/ftclib/gamepad/TriggerReader.java

```diff
@@ -26,21 +31,36 @@ public TriggerReader(GamepadEx gamepad, GamepadKeys.Trigger trigger) {
         this.gamepad = gamepad;
         this.trigger = trigger;
 
-        if (this.gamepad.getTrigger(trigger) > 0.5) {
+        if (this.gamepad.getTrigger(trigger) > this.threshold) {
             currState = true;
         } else {
             currState = false;
         }
 
         lastState = currState;
     }
+    
+    public TriggerReader(GamepadEx gamepad, GamepadKeys.Trigger trigger, double threshold)
+    {
```

Comment:
Make sure your open brackets start on the same line as the method header, we want to maintain the same style throughout all files.